platform/iphone/app_delegate.mm

```diff
@@ -253,11 +271,24 @@ - (void)applicationDidFinishLaunching:(UIApplication*)application {
 	[window makeKeyAndVisible];
 
 	//Configure and start accelerometer
+/*
+  Old accelerometer approach deprecated since IOS 7.0
```

Comment:
I'd say go ahead and remove the obsolete code, I don't think Apple will be reintroducing obsoleted APIs in future iOS versions :)